### PR TITLE
Add mailing list

### PR DESCRIPTION
**Pull request by aakankshaduggal**
Add a link for the mailing list.

### CODE REVIEW

File: docs/meetup-landing-page.md [hunk 1]
```diff
@@ -27,5 +27,5 @@ Watch previous sessions on [Youtube!](https://www.youtube.com/channel/UCe87bwqlG
 
 ## Contact
 
-This meetup is hosted by Red Hat’s AI CoE, part of the Office of the CTO and the Operate First initiative and we can be reached by [slack](https://operatefirst.slack.com/archives/C02KY881CJE) or via the email list. 
+This meetup is hosted by Red Hat’s AI CoE, part of the Office of the CTO and the Operate First initiative and we can be reached by [slack](https://operatefirst.slack.com/archives/C02KY881CJE) or via the [email list](https://lists.operate-first.cloud/admin/lists/community.lists.operate-first.cloud/).
```
Comment (MichaelClifford) [inline docs/meetup-landing-page.md:30]:
```suggestion
This meetup is hosted by Red Hat’s AI CoE, part of the Office of the CTO and the [Operate First initiative](https://www.operate-first.cloud/) and we can be reached by [slack](https://operatefirst.slack.com/archives/C02KY881CJE) or via the [email list](https://lists.operate-first.cloud/admin/lists/community.lists.operate-first.cloud/). 
```

File: docs/meetup-landing-page.md [hunk 2]
```diff
@@ -27,5 +27,5 @@ Watch previous sessions on [Youtube!](https://www.youtube.com/channel/UCe87bwqlG
 
 ## Contact
 
-This meetup is hosted by Red Hat’s AI CoE, part of the Office of the CTO and the Operate First initiative and we can be reached by [slack](https://operatefirst.slack.com/archives/C02KY881CJE) or via the email list. 
+This meetup is hosted by Red Hat’s AI CoE, part of the Office of the CTO and the Operate First initiative and we can be reached by [slack](https://operatefirst.slack.com/archives/C02KY881CJE) or via the [email list](https://lists.operate-first.cloud/admin/lists/community.lists.operate-first.cloud/). 
 This project is maintained as part of the [Operate First](https://www.operate-first.cloud/) and AIOps teams in Red Hat’s AI CoE as part of the Office of the CTO. More information can be found at https://www.operate-first.cloud/.
```
Comment (MichaelClifford) [inline docs/meetup-landing-page.md:31]:
```suggestion

```

**Review by MichaelClifford — COMMENTED** [review]
Also, let's change "Cloud" to "Operate" in all places. I think Cloud First vs Operate First will get confusing.

**Comment by aakankshaduggal** [comment]
> Also, let's change "Cloud" to "Operate" in all places. I think Cloud First vs Operate First will get confusing.

Sure :+1:

**Pull request merged**